tests/resources/config/config12

```diff
@@ -11,3 +11,10 @@ and other stuff !@#"
 multiline comment # with ;\n special chars \
 and other stuff !@#"  #^^^
       back = "this is \ba phrase"
+      dollar = some $sign
+      multiquotes = !ls  "x" \
+                    ls  "# comment2" \
+                    $HOME
+      multiquotes2 = !ls  "x" \
+                     ls  "\"# comment2" \
+                     $HOME\"
```

Comment:
Oof. I sometimes really wonder how people come up with such config files :)

Reply:
Yes, it seems to work without this change.

Reply:
Don't ask ;)